sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -252,86 +258,88 @@ event1 = CloudEvent(
 
-Use a dict-like representation to send to a topic with custom schema as shown below.
+Use EventGridClient's receive function to receive CloudEvents from a Namespace event subscription. Then try to acknowledge, reject, release or renew the locks. 
 
 ```python
 import os
 import uuid
 import datetime as dt
-from msrest.serialization import UTC
 from azure.core.credentials import AzureKeyCredential
-from azure.eventgrid import EventGridPublisherClient
-
-key = os.environ["CUSTOM_SCHEMA_ACCESS_KEY"]
-endpoint = os.environ["CUSTOM_SCHEMA_TOPIC_HOSTNAME"]
+from azure.eventgrid import EventGridConsumerClient
 
-event = custom_schema_event = {
-    "customSubject": "sample",
-    "customEventType": "sample.event",
-    "customDataVersion": "2.0",
-    "customId": uuid.uuid4(),
-    "customEventTime": dt.datetime.now(UTC()).isoformat(),
-    "customData": "sample data"
-    }
+key = os.environ["EVENTGRID_KEY"]
+endpoint = os.environ["EVENTGRID_ENDPOINT"]
+topic_name = os.environ["EVENTGRID_TOPIC_NAME"]
+sub_name = os.environ["EVENTGRID_EVENT_SUBSCRIPTION_NAME"]
 
 credential = AzureKeyCredential(key)
-client = EventGridPublisherClient(endpoint, credential)
+client = EventGridConsumerClient(endpoint, credential, namespace_topic=topic_name, subscription=sub_name)
 
-client.send(event)
-```
+events = client.receive(max_events=4)
 
-### Consume from storage queue
+for detail in events.value:
+    data = detail.event.data
+    broker_properties = detail.broker_properties
+    if data == "release":
+        release_events.append(broker_properties.lock_token)
+    elif data == "acknowledge":
+        acknowledge_events.append(broker_properties.lock_token)
+    else:
+        reject_events.append(broker_properties.lock_token)
 
-This example consumes a message received from storage queue and deserializes it to a CloudEvent object.
+    # Renew all Locks
+    renew_tokens = e.broker_properties.lock_token
+    renew_result = client.renew_locks(
+        lock_tokens=renew_tokens,
+    )
 
-```python
-from azure.core.messaging import CloudEvent
-from azure.storage.queue import QueueServiceClient, BinaryBase64DecodePolicy
-import os
-import json
 
-# all types of CloudEvents below produce same DeserializedEvent
-connection_str = os.environ['STORAGE_QUEUE_CONN_STR']
-queue_name = os.environ['STORAGE_QUEUE_NAME']
+release_result = client.release(
+    lock_tokens=release_events,
+)
 
-with QueueServiceClient.from_connection_string(connection_str) as qsc:
-    payload =  qsc.get_queue_client(
-        queue=queue_name,
-        message_decode_policy=BinaryBase64DecodePolicy()
-        ).peek_messages()
+ack_result = client.acknowledge(
+    lock_tokens=acknowledge_events,
+)
+
+reject_result = client.reject(
+    lock_tokens=reject_events,
+)
 
-    ## deserialize payload into a list of typed Events
-    events = [CloudEvent.from_dict(json.loads(msg.content)) for msg in payload]
 ```
 
-### Consume from servicebus
+### Legacy EventGrid operations
```

Comment:
wondering if you intentionally left this in here even though it's legacy? Is the plan to try to get people to move away from legacy or no? If so, might be good to remove these in the readme and have a legacy folder in the samples that people can refer to if they really need.